mmhuman3d/utils/transforms.py

```diff
@@ -546,3 +546,144 @@ def _sja_to_aa(sja, R_t, R_t_inv):
         raise ValueError(f'Invalid input R_t_inv shape f{R_t.shape}.')
     t = Compose([_sja_to_aa])
     return t(sja, R_t=R_t, R_t_inv=R_t_inv)
+
+
+def batch_transform_to_camera_frame(global_orient, transl, pelvis, extrinsic):
```

Comment:
Can we also keep `transform_to_camera_frame` as there might be situations that we don't have a batch dim.